pkg/resources/validation/feature/feature.go

```diff
@@ -3,42 +3,67 @@ package feature
 import (
 	"fmt"
 	"net/http"
-	"time"
 
+	"github.com/rancher/webhook/pkg/admission"
 	objectsv3 "github.com/rancher/webhook/pkg/generated/objects/management.cattle.io/v3"
-	"github.com/rancher/wrangler/pkg/webhook"
+	admissionv1 "k8s.io/api/admission/v1"
+	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/utils/trace"
 )
 
-func NewValidator() webhook.Handler {
-	return &featureValidator{}
+var gvr = schema.GroupVersionResource{
+	Group:    "management.cattle.io",
+	Version:  "v3",
+	Resource: "features",
 }
 
-type featureValidator struct{}
+// Validator for validating features.
+type Validator struct{}
 
-func (fv *featureValidator) Admit(response *webhook.Response, request *webhook.Request) error {
+// GVR returns the GroupVersionKind for this CRD.
+func (v *Validator) GVR() schema.GroupVersionResource {
+	return gvr
+}
+
+// Operations returns list of operations handled by this validator.
+func (v *Validator) Operations() []admissionregistrationv1.OperationType {
+	return []admissionregistrationv1.OperationType{admissionregistrationv1.Update}
+}
+
+// ValidatingWebhook returns the ValidatingWebhook used for this CRD.
+func (v *Validator) ValidatingWebhook(clientConfig admissionregistrationv1.WebhookClientConfig) *admissionregistrationv1.ValidatingWebhook {
+	valWebhook := admission.NewDefaultValidationWebhook(v, clientConfig, admissionregistrationv1.ClusterScope)
+	valWebhook.FailurePolicy = admission.Ptr(admissionregistrationv1.Ignore)
```

Comment:
Was this the failure policy before this? I'm not finding it

Reply:
yeah so, before we had two webhook configurations, one had a failure policy of ignore [here](https://github.com/rancher/webhook/blob/14683a296710e23bbf805a304be0241467d73c58/pkg/server/server.go#L168-#L176) and the rules which it applied to (cluster & features) are defined [here](https://github.com/rancher/webhook/blob/14683a296710e23bbf805a304be0241467d73c58/pkg/server/rules.go#L5-#L29).